app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -658,7 +658,7 @@ public String getFragmentTag() {
     // Only supports java.io.File. TODO: Android Content
     public boolean saveDocument(final boolean forceSaveEmpty) {
         // Document is written iff content has changed
-        if (_isTextChanged && _document != null && _hlEditor != null && isAdded()) {
+        if (_document != null && _hlEditor != null && isAdded()) {
```

Comment:
Don't rely on external isTextChanged as we don't test this with every change. Save will check this internally anyway.